app/assets/stylesheets/autocomplete.scss

```diff
@@ -0,0 +1,71 @@
+// for autocomplete suggestions
+
+dd.dd-autocomplete-suggestions {
+  position: relative;
+}
+
+.autocomplete-suggestions-container {
+  display: block;
+
+  // coordinates
+  position: absolute;
+  width: 440px;
+  max-width: 100%;
+  max-height: 180px;
+  top: 2px;
+  z-index: 1000;
+
+  // look & feel
+  background: #fff;
+  border: 1px solid #c1c1c1;
+  border-radius: 3px;
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.3);
+
+  // behaviors
+  overflow-x: hidden;
+  overflow-y: scroll;
+
+  // list
```

Comment:
This comment isn't really needed, it doesn't add anymore context to what is going on.